src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -375,6 +375,23 @@ void Convolution::getSupportedDescriptors() {
 
     withBiases = getOriginalInputsNumber() == 3;
 
+    // should remove after binary postops performance issue resolved
+    // heuristics: if it's int8 model and it has binary post ops we will not use brgconv
+    auto tryDisableBrgInt8 = [&]() {
```

Comment:
looks like we don't need lambda at all given it is called one time right after definition.

Reply:
Removed.